jest.config.js

```diff
@@ -38,6 +38,7 @@ const projects = [
     displayName: 'ReactDOM',
     setupFiles: ['<rootDir>/scripts/testSetup.js'],
     testEnvironment: 'jsdom',
+    transformIgnorePatterns: ['/node_modules/(?!@babel/runtime)'],
```

Comment:
is this to transform just babel-runtime, or only that?

Reply:
This just transforms babel runtime. With default all of node modules was ignored